ansible/module_utils/hashivault.py

```diff
@@ -42,6 +44,25 @@ def hashivault_init(argument_spec, supports_check_mode=False, required_if=None,
     return module
 
 
+def hashivault_normalize_from_doc(module, options, documentation):
+    desired_state = {}
+    for key, value in options.items():
+        config_type = documentation.get(key, {}).get('type')
+        if config_type is None:
+            module.warn('Unknown option "{}". Make sure this is not a typo, if it is not, please open an '
+                'issue at https://github.com/TerryHowe/ansible-modules-hashivault/issues.'.format(key))
```

Comment:
```suggestion
            module.warn('Unknown option "{}". Make sure this is not a typo, if it is not, please open an '
                                   'issue at https://github.com/TerryHowe/ansible-modules-hashivault/issues.'.format(key))
```
maybe? I don't know, did not test. Lint is both a pita and nice to have